service_windows.go

```diff
@@ -0,0 +1,233 @@
+package main
```

Comment:
Are we allows an explicit windows build tag? I realise it's not strictly needed.

Reply:
I think it's better to keep the tag in the filename rather than adding it as a comment as this way, the whole file doesn't have to be processed when building on non-windows platforms.